src/default-xpring-client.ts

```diff
@@ -31,7 +62,29 @@ class DefaultXpringClient implements XpringClientDecorator {
    * @returns A `BigInt` representing the number of drops of XRP in the account.
    */
   public async getBalance(address: string): Promise<BigInt> {
-    throw new Error(XpringClientErrorMessages.unimplemented);
+    if (!Utils.isValidXAddress(address)) {
+      return Promise.reject(
+        new Error(XpringClientErrorMessages.xAddressRequired)
+      );
+    }
+
+    const account = new AccountAddress();
+    account.setAddress(address);
```

Comment:
This feels like a weird pattern. Why not `const account = new AccountFromAddress(address)`?

Reply:
`AccountAddress` is a generated class: https://github.com/ripple/rippled/blob/develop/src/ripple/proto/rpc/v1/amount.proto#L19

The JavaScript API for protocol buffers doesn't offer a builder or other pattern to assist here, hence the awkward syntax. I could wrap this into a helper if you think that's better?

Reply:
Hmmm. I think a helper would be beneficial, but I don't think it's necessary for this branch.